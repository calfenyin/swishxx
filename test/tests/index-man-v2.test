index | | -e man:*.[1-9] -r -v2 | . | 0
