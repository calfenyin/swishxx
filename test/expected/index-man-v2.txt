.

index: ranking index...
index: writing index...

index: done:
  9 indexed
  7629 words, 3014 indexed, 755 unique

